src/UserStory/Entries.js

```diff
@@ -0,0 +1,112 @@
+import React, { Component } from 'react'
+import { Link } from 'react-router-dom'
+import validate from 'validate.js'
+
+const initialState = {
+    storyFields: { date:"", owner:"", usId:"", title:"", question:"", note:"" },
+    fieldErrors: { dateError:"", ownerError:"", usIdError:"", titleError:"" }
+}
+
+const constraints = {
+    date: {
+        presence: {
+            allowEmpty : false
+        }
+    },
+    owner: {
+        presence: {
+            allowEmpty : false
+        }
+    },
+    usId: {
+        presence: {
+            allowEmpty : false
+        }
+    },
+    title: {
+        presence: {
+            allowEmpty : false
+        }
+    }
+}
+
+class Entries extends Component {
+    constructor(props){
+        super(props)
+        this.state = initialState;
+    }
+
+    validate() {
+        const error = validate(this.state.storyFields, constraints)
+
+        if(error){ 
+            this.setState({
+                dateError: error.date,
+                ownerError: error.owner,
+                usIdError: error.usId,
+                titleError: error.title
+            })
+            return false
+        }
+        return true
+    }
+
+    handleSubmit = (event) => {
+        event.preventDefault()
+        const isValid = this.validate()
+        if (isValid) {
+            console.log("Data taken, reset field")
+            this.setState(initialState);
+        } else {
+            console.log(this.state.error, "Incomplete form, see error")
+        }
+    }
+
+    handleInputChange = (event) => {
+        const { id, value } = event.target;
+        this.setState((prevState) => {
+            const { storyFields, fieldErrors } = prevState;
+            return { storyFields: { ...storyFields,  [id]: value}, fieldErrors:{ ...fieldErrors, [id]: value}};
+        });
+    }
+    
+    render () {
+        return (
+             <div className="container">
+            <h4 className="center">Input User Story</h4>
+
+                <div className="input-field center col s6">
+                    <form id="form" onSubmit= {this.handleSubmit}>
+                        <label for="date">Date:</label>
+                        {this.state.dateError && <div style={{color:"red"}}>{this.state.dateError}</div> }
+                        <input placeholder="Date" id="date" type="date" className="validate" onInput={this.handleInputChange} value={this.state.storyFields.date}/>
+                         
```

Comment:
As you can see, the pattern of the inputs is repeated very times. Try to slit it into the new component, then reuse it here.